src/mats_l1_processing/read_parquet_functions.py

```diff
@@ -278,10 +278,16 @@ def read_ccd_data_in_interval(
     )
     if filter != None:
         for variable in filter.keys():
-            filterlist &= (
-                (ds.field(variable) >= filter[variable][0])
-                & (ds.field(variable) <= filter[variable][1])
-            )
+            if type(filter[variable]) == list and len(filter[variable]) == 2:
+                filterlist &= (
+                    (ds.field(variable) >= filter[variable][0])
+                    & (ds.field(variable) <= filter[variable][1])
+                )
+            elif type(filter[variable]) in [int,str,float,bool]:
```

Comment:
What about datetimes? I'm not 100%, but I think there are some of those that you may want to filter on as well.

Reply:
You always filter on datetime to start with, or do you want to have a datetime list?

Reply:
There are also ramses times and maybe others in the dataset. Maybe you will never want to filter on those, but you're probably right: it makes little sense to look for a particular exact `datetime` match.

Reply:
(You are the end-users, nod me, so I don't have an opinion either way, but I thought I should point out what could have been an oversight...;))

Reply:
yes, but I think you can filter on expdate by the normal filters